qhub/initialize.py

```diff
@@ -40,6 +40,7 @@
         "jupyterlab": f"quansight/qhub-jupyterlab:{qhub_image_tag}",
         "dask_worker": f"quansight/qhub-dask-worker:{qhub_image_tag}",
         "dask_gateway": f"quansight/qhub-dask-gateway:{qhub_image_tag}",
+        "conda_store": "quansight/qhub-conda-store:sha-a24165f",
```

Comment:
Should be `quansight/qhub-conda-store:{qhub_image_tag}`

Reply:
Yes I know, I only did that because the CI tests were failing.